### ISSUE TITLE
mwc713 integration

### ISSUE THREAD

**Issue by bayk** [mwcproject/swapsite#22]
When mwc-wallet will be done, similar command need to be added into mwc713.

**Comment by hcasalet** [comment]
The challenging part is to figure out how to call mwc-wallet from mwc713 with WalletConfig unknown.

**Comment by bayk** [comment]
Yes, we can't use WalletConfig t function signatures that mwc713 calling.  Signature should get the config fields that are really user. There are not many of them
